src/components/ComboBox/MultiSelectedItemInner.tsx

```diff
@@ -93,7 +93,7 @@ const Wrapper = styled.div<{ themes: Theme; disabled?: boolean }>`
 const ItemLabel = styled.div<{ enableEllipsis?: boolean; themes: Theme }>`
   ${({ enableEllipsis, themes: { border, spacingByChar } }) => {
     return css`
-      flex: 1;
+      flex: 1 1 0;
```

Comment:
flex-basis は初期値 `auto` なので、こうかしら。

```suggestion
      flex: 1 1 auto;
```

Reply:
デフォルトではなく、 Chrome や Firefox が `flex: 1` をこう解釈するのでそのように置き換えています。

Reply:
あああ、そうでした! 🙏